core/scripts/vrfv2/testnet/main.go

```diff
@@ -832,6 +834,108 @@ func main() {
+	case "wrapper-deploy":
+		cmd := flag.NewFlagSet("wrapper-deploy", flag.ExitOnError)
+		linkAddress := cmd.String("link-address", "", "address of link token")
+		linkETHFeedAddress := cmd.String("link-eth-feed", "", "address of link-eth-feed")
+		coordinatorAddress := cmd.String("coordinator-address", "", "address of the vrf coordinator v2 contract")
+		helpers.ParseArgs(cmd, os.Args[2:], "link-address", "link-eth-feed", "coordinator-address")
+		wrapperDeploy(e,
+			common.HexToAddress(*linkAddress),
+			common.HexToAddress(*linkETHFeedAddress),
+			common.HexToAddress(*coordinatorAddress))
+	case "wrapper-configure":
+		cmd := flag.NewFlagSet("wrapper-configure", flag.ExitOnError)
+		wrapperAddress := cmd.String("wrapper-address", "", "address of the VRFV2Wrapper contract")
+		wrapperGasOverhead := cmd.Uint("wrapper-gas-overhead", 50_000, "amount of gas overhead in wrapper fulfillment")
+		coordinatorGasOverhead := cmd.Uint("coordinator-gas-overhead", 52_000, "amount of gas overhead in coordinator fulfillment")
+		wrapperPremiumPercentage := cmd.Uint("wrapper-premium-percentage", 25, "gas premium charged by wrapper")
+		keyHash := cmd.String("key-hash", "", "the keyhash that wrapper requests should use")
+		maxNumWords := cmd.Uint("max-num-words", 10, "the keyhash that wrapper requests should use")
+		helpers.ParseArgs(cmd, os.Args[2:], "wrapper-address", "key-hash")
+
+		wrapperConfigure(e,
+			common.HexToAddress(*wrapperAddress),
+			*wrapperGasOverhead,
+			*coordinatorGasOverhead,
+			*wrapperPremiumPercentage,
+			*keyHash,
+			*maxNumWords)
+	case "wrapper-consumer-deploy":
+		cmd := flag.NewFlagSet("wrapper-consumer-deploy", flag.ExitOnError)
+		linkAddress := cmd.String("link-address", "", "address of link token")
+		wrapperAddress := cmd.String("wrapper-address", "", "address of the VRFV2Wrapper contract")
+		helpers.ParseArgs(cmd, os.Args[2:], "link-address", "wrapper-address")
+
+		wrapperConsumerDeploy(e,
+			common.HexToAddress(*linkAddress),
+			common.HexToAddress(*wrapperAddress))
+	case "wrapper-consumer-request":
+		cmd := flag.NewFlagSet("wrapper-consumer-request", flag.ExitOnError)
+		consumerAddress := cmd.String("consumer-address", "", "address of wrapper consumer")
+		cbGasLimit := cmd.Uint("cb-gas-limit", 100_000, "request callback gas limit")
+		confirmations := cmd.Uint("request-confirmations", 3, "request confirmations")
+		numWords := cmd.Uint("num-words", 1, "num words to request")
+		helpers.ParseArgs(cmd, os.Args[2:], "consumer-address")
+
+		consumer, err := vrfv2_wrapper_consumer_example.NewVRFV2WrapperConsumerExample(
+			common.HexToAddress(*consumerAddress), e.Ec)
+		helpers.PanicErr(err)
+
+		tx, err := consumer.MakeRequest(e.Owner, uint32(*cbGasLimit), uint16(*confirmations), uint32(*numWords))
+		helpers.PanicErr(err)
+		helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+	case "wrapper-universe-deploy":
+		cmd := flag.NewFlagSet("wrapper-universe-deploy", flag.ExitOnError)
```

Comment:
Can extract this into it's own method (like `deployWrapperUniverse`) similar to `deployUniverse` for VRF v2.